lib/sinon/sandbox.js

```diff
@@ -445,8 +446,13 @@ function Sandbox(opts = {}) {
         return spy;
     }
 
-    sandbox.spy = function spy() {
-        const createdSpy = sinonSpy.apply(sinonSpy, arguments);
+    sandbox.spy = function spy(object, property, types) {
+        const createdSpy = sinonSpy.apply(sinonSpy, [
+            object,
+            property,
+            types,
+            sandbox,
```

Comment:
If you are passing in the sandbox as the context, then what is the need for the global context object?